pkg/proc/native/proc_linux.go

```diff
@@ -362,14 +364,12 @@ func (dbp *nativeProcess) trapWaitInternal(pid int, options trapWaitOptions) (*n
 			th.os.delayedSignal = int(status.StopSignal())
 			th.os.running = false
 			return th, nil
-		} else {
-			if err := th.resumeWithSig(int(status.StopSignal())); err != nil {
-				if err == sys.ESRCH {
-					dbp.postExit()
-					return nil, proc.ErrProcessExited{Pid: dbp.pid}
-				}
-				return nil, err
+		} else if err := th.resumeWithSig(int(status.StopSignal())); err != nil {
```

Comment:
For deepsource.

Reply:
As far as I am concerned, you shouldn't worry about deepsource errors if they aren't in code you wrote.